### PR TITLE
[homework#23]-BE-1-intro by Darya Shimanskaya

### PR DESCRIPTION
**Pull request by vastya**

### CODE REVIEW

File: homeworks/darya-shimanskaya/23-BE-1-intro/server.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const http = require('http')
+const fs = require('fs')
+const { URL } = require('url')
+
+if (fs.existsSync('./.env')) {
+  process.env.PORT = (fs.readFileSync('./.env')).toString().split(':')[1].trim()
```
Comment (shov) [inline homeworks/darya-shimanskaya/23-BE-1-intro/server.js:6]: .env files usually use =
we cannot be sure that .env contains PORT only

File: homeworks/darya-shimanskaya/23-BE-1-intro/server.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+const PORT = process.env.PORT || 3000
+
+const server = http.createServer(  (req, res) => {
```
Comment (DmitryMishin) [inline homeworks/darya-shimanskaya/23-BE-1-intro/server.js:11]: please remove 2 spaces

File: homeworks/darya-shimanskaya/23-BE-1-intro/server.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    res.writeHead(200, {'Content-Type': 'text/js'});
+    const stream = fs.createReadStream(__dirname + '/assets/main.js')
+    stream.pipe(res)
+  } else if (path.pathname === '/assets/main.css') {
+    res.writeHead(200, {'Content-Type': 'text/css'});
+    const stream = fs.createReadStream(__dirname + '/assets/main.css')
+    stream.pipe(res)
+  } else {
+    res.writeHead(404, {})
+    res.end("page not found")
```
Comment (DmitryMishin) [inline homeworks/darya-shimanskaya/23-BE-1-intro/server.js:28]: use either single quotes or double quotes

File: homeworks/darya-shimanskaya/23-BE-1-intro/server.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  }
+});
+
+server.listen( PORT, 'localhost', () => {
```
Comment (DmitryMishin) [inline homeworks/darya-shimanskaya/23-BE-1-intro/server.js:32]: are you sure you need this space?

**Review by DmitryMishin — CHANGES_REQUESTED** [review]

File: homeworks/darya-shimanskaya/23-BE-1-intro/server.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+const server = http.createServer(  (req, res) => {
+  const path = new URL(req.url, `http://${req.headers.host}`)
+
+  if (path.pathname === '/') {
+    res.writeHead(200, {'Content-Type': 'text/html'});
+    const stream = fs.createReadStream(__dirname + '/index.html')
+    stream.pipe(res)
+  } else if (path.pathname === '/assets/main.js') {
+    res.writeHead(200, {'Content-Type': 'text/js'});
```
Comment (DmitryMishin) [inline homeworks/darya-shimanskaya/23-BE-1-intro/server.js:19]: remove the semicolon here and in a few more places or add them everywhere

**Pull request merged**